content/blog/2018-12-28-subquery.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Subquery
+created_at: 2018-12-28
```

Comment:
This should only go out after 2.7.0, otherwise there'll be confusion.

Reply:
I was thinking this would be like a teaser before the actual release. If the plan is to post it after 2.7.0 release, then I will change the date after the release. I guess this PR will be on hold till then.

Reply:
From past experience, we'd get a fair few users wondering why it doesn't work because it's not out yet.